### PR TITLE
Switching to be consistent with RE api time-travel nodes.

### PR DESCRIPTION
**Pull request by slebras**
Should fix bug for now, but needs more addressing

* [x] I incremented the version in the `VERSION` file
* [x] I updated `CHANGELOG.md`
* [x] I added test coverage for any changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - adding narrative_config.yaml spec file for separating narrative indexing.
 
+## [1.9.19] - 2021-02-26
```
Comment (jayrbolton) [inline CHANGELOG.md:12]: Can you move this under the [Unreleased] header, then we don't need a version here

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.9.18
+1.9.19
```
Comment (jayrbolton) [inline VERSION:1]: Can revert

Comment (slebras) [reply VERSION]: reverted

File: CHANGELOG.md (hunk 1 above)
Comment (slebras) [reply CHANGELOG.md]: moved

**Pull request merged**